quizcon/templates/main/lti_assignment.html

```diff
@@ -4,49 +4,136 @@
 {% block title %}{{quiz.title|truncatewords:3}}{% endblock %}
 
 {% load coursetags static %}
-
+{% block topnavbar %}
+{% if is_faculty %}
+    <nav class="navbar navbar-expand-lg navbar-dark">
+        <div class="container-fluid">
+            <a class="navbar-brand" href="#"><strong><span class="text-light">QuizCon</span></strong></a>
```

Comment:
Update the href to navigate to the QuizCon dashboard, by setting up an absolute url w/current domain. Also pop it into a new browser tab.

`<a class="nav-link" href="{{ request.scheme }}://{{ request.get_host }}{% url url_name %}"
    target="_blank" rel="noopener noreferrer">`

Reply:
@sdreher Got it!